simpleviewinc/re-theme
# Re theme react app fix

**Ticket**: [issue_id](https://jira.simpleviewtools.com/browse/ZEN-247)


## Context

* I noticed a small setup bug, specifically with the react app portion of `re-theme`

* on a clean repo, `yarn build:app` fails with error: `re-theme` cannot be found in `app/node_modules`
    * due to this dependency: `"re-theme": "file:node_modules/re-theme"`
* `yarn install` also fails in `/app` because of this


## Goal

* be able to run the react app locally without any issue on a fresh clone
    * via `yarn build:app`

## Updates

* `app/.npmrc`
    * so we can install the gh package initially

* Updated `re-theme` imports to `@simpleviewinc/re-theme`



## Testing
* pull this branch
* remove `node_modules` && `yarn.lock` in both the root and `/app`
* run `yarn install` in root
* run `yarn app` in root
    * verify that it successfully install the dependencies/node_modules in `/app`
* run `yarn app` again
    * verify that the app launches
* Spot check the app and ensure that it works the same as before
